client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.html

```diff
@@ -0,0 +1,80 @@
+<div class="text-center"
+     data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div class="row">
+    <div class="col-md-8"
+         data-ng-if="!$ctrl.loading">
+
+        <h2 class="oui-heading_1"
+            data-translate="dedicatedCloud_tab_drp"></h2>
+        <oui-collapsible class="ml-0 mb-5"
+                         data-aria-label="{{:: 'dedicatedCloud_datacenter_drp_subtitle' | translate }}"
+                         data-heading="{{:: 'dedicatedCloud_datacenter_drp_subtitle' | translate }}">
+            <span data-translate="dedicatedCloud_datacenter_drp_details"></span>
+        </oui-collapsible>
+
+        <oui-message class="mb-5"
+                     data-type="warning"
+                     data-ng-if="($ctrl.drpInformations.hasDatacenterWithoutHosts || $ctrl.isDisablingDrp)
+                         && $ctrl.drpInformations.state !== $ctrl.DEDICATEDCLOUD_DATACENTER_DRP_STATUS.deliveredOrProvisionning">
+            <span data-translate="dedicatedCloud_datacenter_drp_warning"
+                  data-ng-if="$ctrl.drpInformations.hasDatacenterWithoutHosts"></span>
+            <span data-translate="dedicatedCloud_datacenter_drp_confirm_delete_drp_success"
+                  data-ng-if="$ctrl.isDisablingDrp"></span>
+        </oui-message>
+
+        <div data-ng-if="$ctrl.$stateParams.currentStep == null">
+            <span data-translate="dedicatedCloud_tab_drp_choice_plan_type"></span>
+            <div class="row d-md-flex mt-3">
+                <div class="col-lg-6">
+                    <oui-select-picker class="h-100"
+                                       data-name="ovh"
+                                       data-model="$ctrl.selectedDrpType"
+                                       data-label="{{:: 'dedicatedCloud_datacenter_ovh_choice' | translate }}"
+                                       data-values="[{ id: 'ovh', name: ('dedicatedCloud_datacenter_ovh_choice' | translate) }]"
+                                       data-variant="light"
+                                       data-disabled="$ctrl.drpInformations.hasDatacenterWithoutHosts || $ctrl.isDisablingDrp">
+                        <oui-select-picker-section>
+                            <figure>
+                                <img class="h-100 w-100"
+                                     src="images/drp/zerto-two-pcc.svg"
+                                     alt="">
```

Comment:
No `alt` description ?

Reply:
Yes, the `oui-select-picker` component will generate a `span` element that contains the name of the option, so this empty `alt` attribute will prevent screen reader to say anything else than what is described by the option title